Contribute/get-started-setup-local.md

```diff
@@ -1,150 +1,217 @@
 
-    ![GitHub two-factor authentication](./media/contribute-get-started-setup-local/github-2fa.png)
+1. Make sure your current working directory is the cloned repository. For example, running the `pwd` command should output:
 
-    > [!Note]
-    > Your credentials will be saved and used to authenticate future GitHub requests. You only need to do this authentication once per computer. 
+   ```bash
+   /c/Users/<Username>/Documents/Docs/<Repo-Name>
+   ```
 
-4. The clone command runs and downloads a copy of the repository files from your fork into a new folder on the local disk. A new folder is made within the current folder. It may take a few minutes, depending on the repository size. You can explore the folder to see the structure once it is finished.
+   If your current working directory is not the cloned repository, change directory (`cd`) to it. You can also test if you are in a repository by running `git status`.
 
-## Configure remote upstream
-After cloning the repository, set up a read-only remote connection to the main repository named **upstream**. You use the upstream URL to keep your local repository in sync with the latest changes made by others. The **git remote** command is used to set the configuration value. You use the **fetch** command to refresh the branch info from the upstream repository.
+2. Add the Microsoft repository as an upstream repository.
 
-1. If you're using **Git Credential Manager**, use the following commands. Replace the \<repo\> and \<organization\> placeholders.
    ```bash
-   cd <repo>
-   git remote add upstream https://github.com/<organization>/<repo>.git
-   git fetch upstream
+   git remote add upstream https://github.com/MicrosoftDocs/<Repo-Name>.git
    ```
 
-2. View the configured values and confirm the URLs are correct. Ensure the **origin** URLs point to your personal fork. Ensure the **upstream** URLs point to the main repository, such as MicrosoftDocs or Azure. 
+3. Validate that the remote URLs are correct.
+
    ```bash
-   git remote -v 
+   git remote -v
    ```
 
-   Example remote output is shown. A fictitious git account named MyGitAccount is configured with a personal access token to access the repo azure-docs:
+   The **origin** URL entires should be your fork URL, while the **upstream** URL entires should be the Microsoft repository URL.
+
    ```output
-   origin  https://github.com/MyGitAccount/azure-docs.git (fetch)
-   origin  https://github.com/MyGitAccount/azure-docs.git(push)
-   upstream        https://github.com/MicrosoftDocs/azure-docs.git (fetch)
-   upstream        https://github.com/MicrosoftDocs/azure-docs.git (push)
+   origin  https://github.com/<Username>/<Repo-Name>.git (fetch)
```

Comment:
Usually, with example output in docs, we'd want to show some literal output. I don't think we'd normally use placeholders on the output. I could go either way here.

Reply:
Got it, if that's what you usually do then I'll replace this with literal output to align with other docs. Is there a specific fictitious username you use?